index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main>
+   <header>
+    <div class="oluchi"><a href="#">Oluchi Theresa</a></div>
+    <div class="port"><a href="#">Portfolio</a></div>
+    <div class="about"><a href="#">About</a></div>
+    <div class="contact"><a href="#">Contact</a></div>
+   </header>
+   <div class="container">
+    <section>
+        <div class="h">
+            <h1>
+                Hi Dear.<br>
+                I am Tessy
+            </h1>
+        </div>
+        <h3>I am a software developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+       </section>
+       <aside>
+        <img src="./images/Illustration-header@2x.png" alt="">
+       <aside>
+   </div>
```

Comment:
- [ ] I noticed the social icons section is missing, please add them 👌

![image](https://user-images.githubusercontent.com/45231276/191676521-98fc54f6-148b-461b-9ebc-1afe6fef35ca.png)